src/middlewares.js

```diff
@@ -233,10 +234,8 @@ export function allowMethodOverride(req, res, next) {
 }
 
 export function handleParseErrors(err, req, res, next) {
-  // TODO: Add logging as those errors won't make it to the PromiseRouter
-  if (err instanceof Parse.Error) {
-    var httpStatus;
-
+  if (err instanceof Parse.Error || err instanceof ParseServer.Error) {
```

Comment:
nit: If ParseServer.Error extends Parse.Error then won't instanceof Parse.Error be enough to check for?